### ISSUE TITLE
What's the structure of the 8.A.json and track.json

### ISSUE THREAD

**Issue by matusbarany04** [SneznyKocur/wocabot#1]
Hi! I was trying to borrow your codebase yet i have encountered some difficulties. Could you please provide a structure of these files, it seems like they are essential for the bot to run. 

```py 
self.word_dictionary = {}
self.tracker_path = "../wocabee archive/track.json"
self.dictionary_path = "../wocabee archive/8.A.json"
self.url = "https://wocabee.app/app"
```

It would be really helpful

**Comment by SneznyKocur** [comment]
ok, i'll also make the values less hardcoded

**Comment by SneznyKocur** [comment]
fixed in [c714173](https://github.com/SneznyKocur/wocabot/commit/c714173e09f54e6d95f46a34407c2940f888b103)?

**Comment by matusbarany04** [comment]
Yes, I think so, thank you for saved hours!
